electron-builder.json

```diff
@@ -10,8 +10,7 @@
   "files": ["**/*", "!**/node_modules/@bitwarden/desktop-native/**/*"],
   "publish": {
     "provider": "s3",
-    "bucket": "public-s3-electron-artifacts",
```

Comment:
We should check if we can use generic provider instead.

Reply:
PR here https://github.com/bitwarden/desktop/pull/1529